app/uk/gov/hmrc/transitmovementsauditing/controllers/AuditController.scala

```diff
@@ -177,7 +177,7 @@ class AuditController @Inject() (
               ),
               detailsRequest.payload
             ),
-            auditType.parent.getOrElse(auditType.name).toString,
+            auditType.parent.map(_.toString).getOrElse(auditType.name),
```

Comment:
this was being inferred as an Object:

```
[warn] /Users/josephbowen/repositories/transit-movements-auditing/app/uk/gov/hmrc/transitmovementsauditing/controllers/AuditController.scala:180:30: a type was inferred to be `Object`; this may indicate a programming error.
[warn]             auditType.parent.getOrElse(auditType.name).toString,
```